tests/test_calendar.py

```diff
@@ -22,18 +22,4 @@ def test_ical_export(django_db_setup, client):
     # The response should return a 200, present a content type of 'text/calendar', and contain an attachment with the slug as its file name.
     assert response.status_code == 200
     assert response['Content-Type'] == 'text/calendar'
-    assert response['Content-Disposition'] == 'attachment; filename={}.ics'.format(event.slug)
-
-@pytest.mark.django_db
-def test_google_calendar_export(django_db_setup, client, mocker):
-    mock_gcal_helper = mocker.patch('nyc.views.google_calendar_export_helper', autospec=True)
-
-    event = Event.objects.all().first()
-    url = reverse('nyc:google_calendar_export', kwargs={'slug': event.slug})
-    response = client.get(url)
-
-    event_arg, _ = mock_gcal_helper.call_args
-    # The event called in google_calendar_export_helper should be the same as the event passed to the view.
-    assert event_arg[0] == event
-    # The response should redirect.
-    assert response.status_code == 302
\ No newline at end of file
+    assert response['Content-Disposition'] == 'attachment; filename={}.ics'.format(event.slug)
```

Comment:
RIP `mock_gcal_helper`! Worth it to test your Gcal URL generator method? Could be as simple as:

```python
e = Event.objects.get(pk=1)
url = create_google_cal_link(e)
assert url == HARD_CODED_URL_WE_EXPECT
```